composer.json

```diff
@@ -28,6 +28,8 @@
     },
     "require": {
         "php"            : ">=5.5.0",
+        "ext-ctype"      : "*",
```

Comment:
Why `ext/ctype`?

Reply:
PhpStorm thinks that it is implicit dependency: you use some function from that extension, but didn't declare this dependency. I don't remember exact function name, unfortunately.

Reply:
Correct, we are using `ctype_digit` here:

https://github.com/hoaproject/Compiler/blob/c620f44deff0b4c2d0c27560a3b0f5e7e376e001/Llk/Llk.php#L291

Requiring an extension just for one call is… too much.

I'm OK to add `ext-ctype` here. However I'm going to open an issue to remove this dependency.